src/cache/cache_test.go

```diff
@@ -80,17 +76,13 @@ func TestBasicLRUSuccess(t *testing.T) {
 	fmt.Printf("TestBasicLRUSuccess ...\n")
 	failed := false
 	misses := 0
-	if CACHE_SIZE > 100 {
-		fmt.Printf("\tignoring, CACHE_SIZE too big\n")
-		return
-	}
 	
-	var cache LRUCache
-	cache.Init(CACHE_SIZE)
+	var cache Cache
+	cache.Init(CACHE_SIZE, LRU)
 
 	for i := 0; i < 2; i++ {
 		for j := 0; j < CACHE_SIZE; j++ {
-			filename := "input/" + strconv.Itoa(j) + ".txt"
+			filename := "../input/" + strconv.Itoa(j) + ".txt"
 			f, _ := os.Create(filename) // ignore error if already exists
```

Comment:
similar to comment above about using strings as the files instead of file objects.

Reply:
See answer above, will change and will need to put more thought into the tests.

Reply:
addressed in ff443cb